sequencing-server/src/lib/batchLoaders/simulatedActivityBatchLoader.ts

```diff
@@ -192,7 +199,14 @@ export function mapGraphQLActivityInstance(
   activityInstance: GraphQLSimulatedActivityInstance<any, any>,
   activitySchema: GraphQLActivitySchema,
 ): SimulatedActivity {
+  if (!activityInstance.duration) {
+    throw new Error('Plan does not have a valid duration');
```

Comment:
This is a single activity instance, not the entire plan, so the message should be updated.

Furthermore, I think we should make sure that a failure here does not fail all expansion, but only the one expansion with the offending activity. Partial success is hugely important for these computer-assisted operator tools as the human skills to perform without them diminish over time and a tool blocking operations is very bad.

Reply:
Removed this for now, I'll create a ticket to fix this later.